_data/meltano/extractors/tap-intercom/ticketswap.yml

```
@@ -0,0 +1,78 @@
+capabilities:
+- about
+- batch
+- catalog
+- discover
+- schema-flattening
+- state
+- stream-maps
+description: A Singer Tap for Intercom, built with the Meltano Tap SDK
```

```suggestion
description: Conversational Relationship Platform
```

This is how the other variants are listed so we can just keep it consistent.